app/code/Magento/Review/Block/Adminhtml/Edit/Form.php

```diff
@@ -85,7 +85,8 @@ protected function _prepareForm()
                         [
                             'id' => $this->getRequest()->getParam('id'),
                             'ret' => $this->_coreRegistry->registry('ret'),
-                            'productId' => $this->getRequest()->getParam('productId')
+                            'productId' => $this->getRequest()->getParam('productId'),
+                            'customerId' => $this->getRequest()->getParam('customerId')
```

Comment:
In case of edit from customer page `productId` will be empty?

Reply:
Hello @orlangur 
I have checked this in case of edit from customer page productId is not require because review edit using review id, productId, and customerId is additional param
here I have attached snapshot of edit form action which clearly show productId is not set then not passed in action URL param.

https://prnt.sc/ne3if0

Reply:
@ravi-chandra3197 I mean, there is no need to add `productId` / `customerId` when it is empty. Please refactor and squash changes.

Reply:
@ravi-chandra3197 why did you mark conversation as resolved?

```
'productId' => $this->getRequest()->getParam('productId'),
```

here we should not add `productId` when it is empty. Same for `customerId`.

Reply:
Hello @orlangur 
I have update PR code as per your suggestion now we also check param before adding array can you review it.